agent/tool-scripts/meta.json

```diff
@@ -0,0 +1,48 @@
+{
+	"transient":{
```

Comment:
Interestingly, I'd imagined it "inside out" with ["node-exporter": {"type":"persistent"}, "blktrace":{"type":"transient"}, ...]

Yours is better as you can 'tool in meta.transient.keys()' which makes a lot more sense, and we can still easily define properties for each.

Reply:
We can dynamically create the convenient way to access the data, too.

Reply:
We absolutely want all behavior to be encapsulated behind a class API, with nobody outside the class (or singleton, since I generally dislike static members) knowing anything about the JSON schema.